pkg/controller/job/job_controller.go

```diff
@@ -1449,6 +1454,9 @@ func (jm *Controller) manageJob(ctx context.Context, job *batch.Job, jobCtx *syn
 			return 0, metrics.JobSyncActionPodsCreated, nil
```

Comment:
above:
- line 1430: `rmAtLeast := active - (wantActive - terminating)`
- line 1450: `if diff := wantActive - terminating - active; diff > 0 {`

Reply:
I think first one makes sense to me but I don't really know where you want second one to go now.